Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -257,24 +257,24 @@ public extension Sequence where Element: Equatable {
     ///        ["h", "e", "l", "l", "o"].contains(["l", "o"]) -> true
     ///
     /// - Parameter elements: array of elements to check.
-    /// - Returns: true if array contains all given items.
+    /// - Returns: true if sequence contains all given items.
     /// - Complexity: _O(m·n)_, where _m_ is the length of `elements` and _n_ is the length of this sequence.
     func contains(_ elements: [Element]) -> Bool {
```

Comment:
I meant this should become as sequence argument as well? 
```suggestion
    func contains<S: Sequence>(_ elements: S) -> Bool where S.Element == Self.Element {
```